src/char/inter.h

```diff
@@ -61,7 +61,11 @@ struct inter_interface {
 };
 
 #ifdef HERCULES_CORE
-extern int party_share_level; ///< Share range for parties.
+#ifndef RENEWAL
+	extern int party_share_level; ///< Share range for parties in Pre-RE mode.
```

Comment:
probably better left in C code only ``party_share_level`` field. This will reduce code a bit.